msteams-platform/sbs-teams-app-test-tool.yml

```diff
@@ -0,0 +1,351 @@
+
+            A dialog appears, where you need to choose yes or no to trust the authors of the files in this folder.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/vsc-trust-authors.png" alt-text="Screenshot shows the dialog to trust or not the authors of the files in this folder.":::
+
+            Now you've successfully created your AI chat bot project workspace.
+
+    ## Take a tour of the bot app source code
+
+    After you finish scaffolding, explore the project directories and files in the Explorer section of the Visual Studio Code.
+    
+    :::image type="content" source="../msteams-platform/assets/images/test-tool/source-code.png" alt-text="Screenshot shows the Teams Toolkit sample bot folder Structure.":::
+    
+    | Folder name | Contents |
+    | --- | --- |
+    | `env/.env.testtool.user` |Configuration file for local environment used by `teamsapp.testtool.yml` to customize the provisioning and deployment rules.  |
+    | `appPackage` | App manifest template files and app icons (color.png and outline.png). |
+    | `appPackage/manifest.json` | App manifest for running the app in local and remote environment.  |
+    |`src/app.js`| Handles business logics for the AI Chat Bot.|
+    |`teamsapp.yml`|This is the main Teams Toolkit project file. The project file defines two primary things:  Properties and configuration Stage definitions. |
+    |`teamsapp.local.yml`|This overrides `teamsapp.yml` with actions that enable local execution and debugging.|
+    |`teamsapp.testtool.yml`|This overrides `teamsapp.yml` with actions that enable local execution and debugging in Test Tool.|
+
+- title: Build and run your AI chat bot app
+  durationInMinutes: 1
+  content: |
+    After you set up your project workspace with Teams Toolkit, build your AI chat bot project. Ensure that you've signed in to your Microsoft 365 account.
+
+    ## Sign in to your Microsoft 365 account
+
+    If you're using a Microsoft 365 developer program tenant, the admin account you set up while registering is your Microsoft 365 account.
+
+    1. Open **Visual Studio Code**.
+
+    1. Select the Teams Toolkit :::image type="icon" source="../msteams-platform/assets/icons/vsc-toolkit.png" border="false"::: icon in the sidebar.
+
+    1. Select **Sign in to M365**. Your default web browser opens to allow you to sign in to the account.
+
+    1. Sign in to your Microsoft 365 account using your credentials.
+
+    1. Close the browser when prompted and return to Visual Studio Code.
+
+    1. Return to Teams Toolkit within Visual Studio Code.
+
+        :::image type="content" source="../msteams-platform/assets/images/test-tool/sideload.png" alt-text="Screenshot showing where to sign in to Microsoft 365 and Azure.":::
+
+    Now you're ready to build the app and debug.
+
+    ## Create Open AI key and Endpoint for your AI chat bot
```

Comment:
Besides the section to use Azure OpenAI API, shall we also cover the instruction about how to debug the AI chat bot with OpenAI's API?

Reply:
@qinezh I've discussed this with Junjie and we decided to add the following Note to use OpenAI's API key to debug and added link to OpenAI API doc.

Note: You can also get OpenAI API key to debug your AI chat bot. For more information, see [Setup your API key](https://platform.openai.com/docs/quickstart/step-2-setup-your-api-key).